pavement.py

```diff
@@ -119,5 +119,5 @@ def release():
     sh("git tag -s '%(name)s-%(version)s' -m 'Version bump to %(version)s'" % {'name': NAME, 'version': VERSION})
     sh("git push --tags")
     sh("git push")
-    sh("python setup.py register sdist upload")
-    sh("python setup.py bdist_wheel upload")
+    sh("python setup.py register sdist bdist_wheel")
```

Comment:
@honzajavorek is right and we don't need register. I'm doing [similar](https://github.com/kylef/maintain/blob/master/maintain/release/python.py#L38-L41):

```python
        version = self.determine_current_version()
        invoke(['python', 'setup.py', 'sdist', 'bdist_wheel'])
        invoke(['twine', 'upload', 'dist/*{}*'.format(version)])
```

Reply:
I wasn't right, because first I made @abtris to put `register` back 😄